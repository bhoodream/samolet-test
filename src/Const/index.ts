export const REGION_ID_KEY = 'kopuk';
export const SHOW_ITEMS_COUNT = 8;
export enum SORT_MODES {
    territory,
    subscribers,
    libraries,
}
