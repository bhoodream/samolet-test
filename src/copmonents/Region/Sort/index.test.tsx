import renderer from 'react-test-renderer';
import RegionsSort from './index';

import { SORT_MODES } from '../../../Const/index';

it('renders correctly', () => {
    const tree = renderer
        .create(<RegionsSort mode={SORT_MODES.libraries} onChange={() => {}} />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
