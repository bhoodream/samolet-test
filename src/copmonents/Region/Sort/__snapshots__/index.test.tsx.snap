// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-select ant-select-lg ant-select-single ant-select-show-arrow"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="ant-select-selector"
    onClick={[Function]}
    onMouseDown={[Function]}
  >
    <span
      className="ant-select-selection-search"
    >
      <input
        aria-activedescendant="undefined_list_0"
        aria-autocomplete="list"
        aria-controls="undefined_list"
        aria-haspopup="listbox"
        aria-owns="undefined_list"
        autoComplete="off"
        className="ant-select-selection-search-input"
        onChange={[Function]}
        onCompositionEnd={[Function]}
        onCompositionStart={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onPaste={[Function]}
        readOnly={true}
        role="combobox"
        style={
          Object {
            "opacity": 0,
          }
        }
        type="search"
        unselectable="on"
        value=""
      />
    </span>
    <span
      className="ant-select-selection-item"
    >
      <div
        className="ant-space ant-space-horizontal ant-space-align-center"
        style={Object {}}
      >
        <div
          className="ant-space-item"
          style={
            Object {
              "marginRight": 8,
            }
          }
        >
          <span
            aria-label="bank"
            className="anticon anticon-bank"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="bank"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M894 462c30.9 0 43.8-39.7 18.7-58L530.8 126.2a31.81 31.81 0 00-37.6 0L111.3 404c-25.1 18.2-12.2 58 18.8 58H192v374h-72c-4.4 0-8 3.6-8 8v52c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-52c0-4.4-3.6-8-8-8h-72V462h62zM512 196.7l271.1 197.2H240.9L512 196.7zM264 462h117v374H264V462zm189 0h117v374H453V462zm307 374H642V462h118v374z"
              />
            </svg>
          </span>
        </div>
        <div
          className="ant-space-item"
          style={Object {}}
        >
          По библиотекам
        </div>
      </div>
    </span>
  </div>
  <span
    aria-hidden={true}
    className="ant-select-arrow"
    onMouseDown={[Function]}
    style={
      Object {
        "WebkitUserSelect": "none",
        "userSelect": "none",
      }
    }
    unselectable="on"
  >
    <span
      aria-label="down"
      className="anticon anticon-down ant-select-suffix"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="down"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
        />
      </svg>
    </span>
  </span>
</div>
`;
